setup.py

```diff
@@ -281,6 +281,17 @@ def generate_level(self, prefix, parser, Archiver):
                     self.write_options(parser, doc)
                     doc.write("\n\nDescription\n~~~~~~~~~~~\n")
                     doc.write(epilog)
+                    doc.write(textwrap.dedent("""
+
+                    .. raw:: html
+
+                        <script type='text/javascript'>
+                        $(document).ready(function () {
+                            /* Remove the "Description" and "Examples" level three entries from the global toc */
+                            $('.toctree-l2 ul').remove();
+                        })
+                        </script>
```

Comment:
Turns out there is. Not really a "native" way, but a bit cleaner than adding a bit of JS.

The reason this is so annoying to do is that the maxdepth parameter of the toctree() template function defaults to the maxdepth of the toctree. The *root* toctree, that is, not the *nearest* toctree relative to the current page. And that's two, and the reason why I put three in there. However, Sphinx gives us `pagename` so a little if is all that is needed to fix that up.